src/Parsing/Impl/Ast/FunctionDefinition.cs

```diff
@@ -19,21 +19,18 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Python.Core;
-using Microsoft.Python.Core.Text;
 
 namespace Microsoft.Python.Parsing.Ast {
     [DebuggerDisplay("{Name}")]
-    public class FunctionDefinition : ScopeStatement, IMaybeAsyncStatement {
+    public class FunctionDefinition : ScopeStatement, IMaybeAsyncStatement, IScopeNode {
         internal static readonly object WhitespaceAfterAsync = new object();
 
         private int? _keywordEndIndex;
 
-        protected Statement _body;
+        private Statement _body;
```

Comment:
This is technically an API break, since anyone could subclass FunctionDefinition. I wouldn't change any of the existing protection levels.

Reply:
OK